extra/prometheus/prometheus.yml

```diff
@@ -122,14 +122,14 @@ scrape_configs:
     metrics_path: /metrics/array
     # Bearer authorization token
     authorization:
-      credentials: a12345bc6-d78e-901f-23a4-56b07b89012
+      credentials: 22222222-2222-2222-2222-222222222222
     params:
       endpoint: ['arrayname02.fqdn.com']
 
     static_configs:
     # purefa openmetrics exporter
     - targets:
-      - 10.0.2.10:9490
+      - purefa-openmetrics-exporter.fqdn.com:9490
       labels:
         location: uk
         site: London
```

Comment:
This should be fr and Paris

Reply:
Well spotted! Resolved in commit de6ba1b4812834ef195edb505000770802cc245b